sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
```

Comment:
I don't think it matters what we call these imperfect gold standards so with whatever you prefer.  My main suggestion is that we explain what we mean by "pyrite" or "bronze" standard in the subsequent sentence.